hare/broker.go

```diff
@@ -139,12 +140,12 @@ func (b *Broker) eventLoop() {
 				// todo: err if chan is full (len)
 				c <- iMsg
 				continue
-			} else { // message arrived before registration
+			} else { // Message arrived before registration
 				futureMsg = true
 			}
 
 			if futureMsg {
-				b.Info("Broker identified future message")
+				b.Info("Broker identified future Message")
```

Comment:
I'd add more info to this message or remove it as it is not very informative.

Reply:
Leftover from debugging. Removed